packages/e2e.test.foundry-sdk-generator/package.json

```diff
@@ -9,34 +9,35 @@
   },
   "exports": {
     ".": {
-      "require": "./build/cjs/index.cjs",
       "browser": "./build/browser/index.js",
       "import": "./build/esm/index.js"
     },
     "./*": {
-      "require": "./build/cjs/public/*.cjs",
       "browser": "./build/browser/public/*.js",
       "import": "./build/esm/public/*.js"
     }
   },
   "scripts": {
-    "check-attw": "monorepo.tool.attw both",
+    "check-attw": "monorepo.tool.attw esm",
```

Comment:
This was a `both` before but didn't need to be. This simplifies logic and reduces build steps